lib/arch/generic/dec_tail.c

```diff
@@ -5,6 +5,7 @@
 		if ((q = base64_table_dec[*c++]) >= 254) {
 			st.eof = BASE64_EOF;
 			// Treat character '=' as invalid for byte 0:
+			ret = 0;
```

Comment:
it's not strictly necessary. I find it easier to understand that way, but, since it's just a personal preference, I removed it.